src/lib_sdl/video.cr

```diff
@@ -95,6 +100,31 @@ lib LibSDL
     RESET_ISOLATION_FLAG    = 0x0008
   end
 
+  struct SDL_Version
+    major : UInt8
+    minor : UInt8
+    patch : UInt8
+  end
+
+  enum SDL_SYSType
+    SDL_SYSWM_UNKNOWN
+    SDL_SYSWM_WINDOWS
+    SDL_SYSWM_X11
+    SDL_SYSWM_DIRECTFB
+    SDL_SYSWM_COCOA
+  end
+
+  struct WM_x11
+    display : Void*
+    window : UInt64
+  end
+
+  struct SDL_WMInfo
+    version : SDL_Version
+    subsystem : SDL_SYSType
+    info : WM_x11 # TODO: Add support for other WM's
```

Comment:
This should be an explicit C union, so we can eventually add support for more systems:

```crystal
struct SDL_WMInfoX11
  display : Void*
  window : UInt32
end

union SDL_WMInfoUnion
  x11: SDL_WMInfoX11
end

struct SDL_WMInfo
  version : SDL_Version
  subsystem : SDL_SYSType
  info: SDL_WMInfoUnion
end
```